sphinxcontrib_robotframework.py

```diff
@@ -139,7 +139,7 @@ def run_robot(app, doctree, docname):
     result = robot.run(robot_file.name, **options)
     if result == 0:
         with open(checksums_filename, 'w') as fp:
-            fp.write(pickle.dumps(checksums + [checksum]))
+            fp.write(pickle.dumps(checksums + [checksum]).decode('latin1'))
```

Comment:
The correct fix is not to decode from latin1 to Unicode before writing, but to open the file in `'wb'` mode instead of `'w'`.

Reply:
Also, remember to open the file for reading in `'b'` mode on line 90.

Reply:
Fixed! Thanks!